docs/conf.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+
+sys.path = ['.', '..'] + sys.path
+
+project = 'AutoGluon-Cloud'
+release = '0.1.1'
+copyright = '2023, All authors. Licensed under Apache 2.0.'
+author = 'AutoGluon contributors'
+
+extensions = [
+    'myst_nb',                       # myst-nb.readthedocs.io
+    'sphinx_copybutton',             # sphinx-copybutton.readthedocs.io
+    'sphinx_design',                 # github.com/executablebooks/sphinx-design
+    'sphinx_inline_tabs',            # sphinx-inline-tabs.readthedocs.io
+    'sphinx_togglebutton',           # sphinx-togglebutton.readthedocs.io
+    'sphinx.ext.autodoc',            # www.sphinx-doc.org/en/master/usage/extensions/autodoc.html
+    'sphinx.ext.autosummary',        # www.sphinx-doc.org/en/master/usage/extensions/autosummary.html
+    'sphinx.ext.napoleon',           # www.sphinx-doc.org/en/master/usage/extensions/napoleon.html
+    'sphinx.ext.viewcode',           # www.sphinx-doc.org/en/master/usage/extensions/viewcode.html
+    'sphinxcontrib.googleanalytics', # github.com/sphinx-contrib/googleanalytics
+    ]
+
+# See https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
+myst_enable_extensions = ['colon_fence', 'deflist', 'dollarmath', 'html_image', 'substitution']
+
+autosummary_generate = True
+numpydoc_show_class_members = False
+
+googleanalytics_id = "UA-96378503-20"
+
+nb_execution_mode = 'force'
+# nb_execution_raise_on_error=True
+nb_execution_timeout = 3600
+nb_merge_streams = True
+
+nb_execution_excludepatterns = ['jupyter_execute']
+
+nb_dirs_to_exec = [os.path.join('tutorials', tag) for tag in tags if os.path.isdir(os.path.join('tutorials', tag))]
+
+if len(nb_dirs_to_exec) > 0:
+    nb_dirs_to_exclude = [dirpath for dirpath, _, filenames in os.walk('tutorials')
+                            if any(map(lambda x: x.endswith('.ipynb'), filenames))
+                               and not dirpath.startswith(tuple(nb_dirs_to_exec))]
+
+    for nb_dir in nb_dirs_to_exclude:
+        nb_execution_excludepatterns.append(os.path.join(nb_dir, '*.ipynb'))
```

Comment:
If you don't need the ability to exclude building some notebooks, I like the idea of getting rid of these lines because they take extra time to understand.

Reply:
I think this is a nice feature and could be useful in the future so I left it here